knot/src/test/kotlin/de/halfbit/knot/KnotEffectTest.kt

```diff
@@ -32,6 +32,44 @@ class KnotEffectTest {
         observer.assertEmpty()
     }
 
+    @Test
+    fun `Knot Effect + null action`() {
+        val actions = PublishSubject.create<Action>()
+        val knot = knot<State, Change, Action> {
+            state {
+                initial = State
+            }
+            changes {
+                reduce { only + null  }
```

Comment:
It looks we need some more tests to get code coverage back:

`{ this + Action.One + null }`
`{ this + Action.One + Action.Two + null }`

and same for primes. Could you add them?

Reply:
Will do!